tchannel/sync/client.py

```diff
@@ -85,6 +88,43 @@ def request(self, *args, **kwargs):
 
         return operation
 
+    def advertise(self, routers, name=None, timeout=None):
+
+        @gen.coroutine
+        def make_request():
+
+            response = yield self.async_client.advertise(
+                router=routers,
+                name=name,
+                timeout=timeout,
+            )
+
+            header = yield response.get_header()
+            body = yield response.get_body()
+
+            result = Response(header, body)
+
+            raise gen.Return(result)
+
+        future = self.threadloop.submit(make_request)
+
+        # we're going to wait 1s longer than advertises
+        # timeout mechanism, so it has a chance to timeout
+        wait_until = timeout or FIRST_ADVERTISE_TIME
```

Comment:
If you're depending on the default value for `FIRST_ADVERTISE_TIME` anyway, might as well use it as the default value for `timeout` before passing it down to the async client.